source/special.tex

```diff
@@ -103,7 +103,7 @@
 The \grammarterm{class-name} shall not be a \grammarterm{typedef-name}. In a constructor
 declaration, each \grammarterm{decl-specifier} in the optional
 \grammarterm{decl-specifier-seq} shall be \tcode{friend}, \tcode{inline},
-\tcode{explicit}, or \tcode{constexpr}.
+or \tcode{constexpr} or an \grammarterm{explicit-specifier}.
```

Comment:
The phrasing here is a bit weird, this parses as "shall be (friend), (inline), or (constexpr or an explicit-specifier)". How about:

> shall be `friend`, `inline`, `constexpr`, or an *explicit-specifier*.

Reply:
Fixed.